digitalearthau/qsub.py

```diff
@@ -345,7 +345,7 @@ def norm_qsub_params(p):
     mem = normalise_mem(p.get('mem', 'small'))
 
     if nodes:
-        mem = int((mem * NUM_CPUS_PER_NODE * 1024 - 512) * nodes)
+        mem = int((mem * NUM_CPUS_PER_NODE * 1024 - 2048) * nodes)
```

Comment:
How do we tune this?
Probably time to take it out into a module level constant or even into a qsub configuration option? That way we can experiment with various values without changing code each time.